js/client-chat.js

```diff
@@ -134,16 +134,10 @@
 			} else if (e.keyCode === 34) { // Pg Dn key
 				this.$chatFrame.scrollTop(this.$chatFrame.scrollTop() + this.$chatFrame.height() - 60);
 			} else if (e.keyCode === 9 && !e.ctrlKey) { // Tab key
-				if (!e.shiftKey) {
-					if (this.handleTabComplete(this.$chatbox, false)) {
-						e.preventDefault();
-						e.stopPropagation();
-					}
-				} else { // Shift + Tab
-					if (this.handleTabComplete(this.$chatbox, true)) {
-						e.preventDefault();
-						e.stopPropagation();
-					}
+				var reverse = e.shiftKey; // Shift+Tab reverses direction
```

Comment:
If you say so, I suppose it can't hurt. I originally didn't because I figured it's not going to be anything other than a bool in the foreseeable future, and even if it were, nothing would break.
But yeah, changed.